mlos_core/mlos_core/optimizers/bayesian_optimizers/smac_optimizer.py

```diff
@@ -0,0 +1,212 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Contains the wrapper class for SMAC Bayesian optimizers.
```

Comment:
```suggestion
Contains the wrapper class for SMAC Bayesian optimizers.
See Also: https://automl.github.io/SMAC3/main/index.html
```

Reply:
Should hopefully provide a quick link to it when the docs are generated from the docstrings.